ui-tests/src/test/resources/features/integrations/template.feature

```diff
@@ -0,0 +1,309 @@
+#@sustainer: mmuzikar@redhat.com
+
+@ui
+@data-mapper
+@template
+Feature: Integration - Database formatted by template to Gmail
+
+  Background: Clean application state
+    Given clean application state
+    Given log into the Syndesis
+    And delete emails from "jbossqa.fuse@gmail.com" with subject "syndesis-template-test"
```

Comment:
it would be better to be consistent with using `And` through the whole file, either use it everywhere where possible or don't use it :grin: